### PR TITLE
Add some documentation about the /q/info endpoint

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -469,6 +469,28 @@ By default, when a Quarkus application starts (in regular or dev mode), it will
...
+* `/q/health`
+* `/q/metrics`
+* `/q/openapi`
+* `/q/info`
+
+but users can also choose to expose one that might present a security risk under a different TCP port using a dedicated xref:management-interface-reference.adoc[management interface].
+
+=== Info endpoint
+
+If the application contains the `quarkus-info` extension, then Quarkus will by default expose the `/q/info` endpoint which contains information about the build, the java version, version control and the operating system and the level of detail of the exposed information is configurable.
```
Comment (rsvoboda) [inline docs/src/main/asciidoc/getting-started.adoc:488]:
```suggestion
If the application contains the `quarkus-info` extension, then Quarkus will by default expose the `/q/info` endpoint which provides information about the build, java version, version control, and operating system. The level of detail of the exposed information is configurable.
```

**Review by rsvoboda — COMMENTED** [review]
Just a small suggestion.

File: docs/src/main/asciidoc/getting-started.adoc [hunk 2]
```diff
@@ -469,6 +469,28 @@ By default, when a Quarkus application starts (in regular or dev mode), it will
 by setting the `-Dquarkus.banner.enabled=false` Java System Property, or by setting the `QUARKUS_BANNER_ENABLED` environment variable to `false`.
 Furthermore, users can supply a custom banner by placing the banner file in `src/main/resources` and configuring `quarkus.banner.path=name-of-file` in `application.properties`.
 
+== Non Application endpoints
+
+Various Quarkus extensions contribute non-application endpoints that provide different kinds of information about the application.
+Examples of such extensions are the xref:smallrye-health.adoc[health], xref:micrometer.adoc[metrics], OpenAPI and info extensions.
```
Comment (rsvoboda) [inline docs/src/main/asciidoc/getting-started.adoc:475]:
```suggestion
Examples of such extensions are the xref:smallrye-health.adoc[health], xref:micrometer.adoc[metrics], xref:openapi-swaggerui.adoc[OpenAPI] and info extensions.
```

Comment (geoand) [reply docs/src/main/asciidoc/getting-started.adoc:475]: Ah nice, I tried to find that one real quick but missed it :)

Comment (geoand) [reply docs/src/main/asciidoc/getting-started.adoc]: Applied, thanks

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**